docs/framework/network-programming/changes-to-the-system-uri-namespace-in-version-2-0.md

```diff
@@ -50,9 +50,9 @@ Several changes were made to the <xref:System.Uri?displayProperty=nameWithType>
   
 -   <xref:System.Uri.IsLoopback%2A> now produces consistent results.  
   
--   The URI "`file:///path`" is no longer translated into "file://path".  
+-   The URI "`file:///path`" is no longer translated into `file://path`.  
   
--   "#" is now recognized as a host name terminator. That is, "http://consoto.com#fragment" is now converted to "http://contoso.com/#fragment".  
+-   "#" is now recognized as a host name terminator. That is, `http://consoto.com#fragment` is now converted to `http://contoso.com/#fragment`.  
```

Comment:
should the # here be escaped `\#`

Reply:
I dont think # needs to be escaped. # only would need escape if it was at the start of a line.